jobs/haproxy/spec

```diff
@@ -704,7 +704,15 @@ properties:
   ha_proxy.backend_config:
     description: |
       Raw HAProxy config that will be added to the default HTTP + routed HTTP backend definitions, provided either as a multiline text blob or as an array of lines.
-
+  ha_proxy.backend_config_targeted:
+      description: |
+        Config property, customizable for various protocols, with different connection reuse options, that will be added to the default HTTP and Routed HTTP backends.
```

Comment:
As I understood the property is not only about http-reuse but another properties can be also possible.
I would say: A map of existing backends which enables more fine-grained configuration of the backend.